osquery/tables/system/darwin/keychain.h

```diff
@@ -9,21 +9,12 @@
 
 #pragma once
 
-#include <map>
 #include <set>
 #include <vector>
 
 #include <CoreFoundation/CoreFoundation.h>
 #include <Security/Security.h>
 
-#include <openssl/x509v3.h>
-#include <openssl/bn.h>
-#include <openssl/asn1.h>
-#include <openssl/x509.h>
-#include <openssl/x509_vfy.h>
-#include <openssl/pem.h>
-#include <openssl/bio.h>
```

Comment:
I think some of these are missing and might be required, probably in `openssl_utils.h` or `certificates.mm` and probably why the macOS build a failing

Reply:
> I think https://github.com/osquery/osquery/blob/master/osquery/tables/system/tests/darwin/certificates_tests.cpp will also need to be updated!

I did a git clean and lost my `-DOSQUERY_BUILD_TESTS=true` and only noticed when I pushed :-1: